components/config/prefs.rs

```diff
@@ -294,6 +294,8 @@ mod gen {
                 webxr: {
                     #[serde(default)]
                     enabled: bool,
+                    #[serde(default)]
+                    test: bool,
```

Comment:
A bit annoying that this does something different than `dom.webvr.test` which renders to a GLWindow. At some point we'll have to come up with consistent naming.

Reply:
I'm also okay with this rendering to a GLWindow in the future, but I'd have to teach the mock backend about it